WordPress/src/main/java/org/wordpress/android/ui/accounts/login/LoginBaseFormFragment.java

```diff
@@ -342,4 +343,8 @@ public void onSiteChanged(SiteStore.OnSiteChanged event) {
 
         onLoginFinished(true);
     }
+
+    public HelpshiftHelper.Tag helpshiftOriginTag() {
```

Comment:
I think `LoginBaseFormFragment` should not implement this and leave it (demand it basically since it's abstract) to be implemented in the subclasses. That way, the actual screen fragments will be forced to define their origin tag. `ORIGIN_UNKNOWN` should not happen anyway as there is no (login) screen with the origin not defined.